src/evict/evict_lru.c

```diff
@@ -2884,13 +2872,13 @@ __wti_evict_app_assist_worker(WT_SESSION_IMPL *session, bool busy, bool readonly
          * Additionally we don't return rollback which could confuse the caller.
          */
         if (__wt_op_timer_fired(session))
-            break;
+            goto err;
```

Comment:
This looks unrelated to the change, this change of flow should be in a different PR.

Reply:
This doesn't change the flow but removes confusion about when "break" means.